src/BSON/PackedArray.c

```diff
@@ -147,22 +160,29 @@ static PHP_METHOD(MongoDB_BSON_PackedArray, getIterator)
 	phongo_iterator_init(return_value, getThis());
 }
 
+static bool php_phongo_packedarray_has(php_phongo_packedarray_t* intern, zend_long index)
+{
+	bson_iter_t iter;
+
+	if (!bson_iter_init(&iter, intern->bson)) {
+		phongo_throw_exception(PHONGO_ERROR_RUNTIME, "Could not initialize BSON iterator.");
```

Comment:
```suggestion
		phongo_throw_exception(PHONGO_ERROR_RUNTIME, "Could not initialize BSON iterator.");
		return false;
```

Same issue as `php_phongo_document_has` above.